.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```

Comment:
@LukasChiama  Add new line here

Reply:
That's a .yml file. White spaces matter in .yml files